sdk/messaging/azservicebus/internal/mgmt.go

```diff
@@ -527,3 +541,13 @@ func CancelScheduledMessages(ctx context.Context, rpcLink RPCLink, seq []int64)
 
 	return nil
 }
+
+// addAssociatedLinkName adds the 'associated-link-name' application
+// property to the AMQP message. Setting this property associates
+// management link activity with a sender or receiver link, which can
+// prevent it from idling out.
+func addAssociatedLinkName(linkName string, msg *amqp.Message) {
+	if linkName != "" {
```

Comment:
Probably not, but the property itself is optional. If we ever end up in a situation where I deliberately want to avoid adding it that's the way to do it.